velox/duckdb/conversion/DuckWrapper.cpp

```diff
@@ -301,19 +383,21 @@ VectorPtr toVeloxVector(
       uint8_t width;
       uint8_t scale;
       type.GetDecimalProperties(width, scale);
+      auto veloxDecimalType = DECIMAL(width, scale);
       switch (type.InternalType()) {
         case PhysicalType::INT16:
-          return convertDecimalToDouble<int16_t>(
-              duckVector, size, veloxType, pool, scale);
+          return convertDuckToVeloxDecimal<DuckShortDecimalConversion, int16_t>(
+              duckVector, veloxDecimalType, size, pool);
         case PhysicalType::INT32:
-          return convertDecimalToDouble<int32_t>(
-              duckVector, size, veloxType, pool, scale);
+          return convertDuckToVeloxDecimal<DuckShortDecimalConversion, int32_t>(
+              duckVector, veloxDecimalType, size, pool);
         case PhysicalType::INT64:
-          return convertDecimalToDouble<int64_t>(
-              duckVector, size, veloxType, pool, scale);
+          return convertDuckToVeloxDecimal<DuckShortDecimalConversion, int64_t>(
+              duckVector, veloxDecimalType, size, pool);
         case PhysicalType::INT128:
-          return convertDecimalToDouble<hugeint_t>(
```

Comment:
Can we remove convertDecimalToDouble now?

Reply:
We can remove it. I am wondering should it be part of another PR?

Reply:
If the usage is removed in this PR, the function itself should be removed in the same PR as well.